Modules/_bz2module.c

```diff
@@ -8,6 +8,58 @@
 #include <bzlib.h>
 #include <stdio.h>
 
+// Blocks output buffer wrappers
+#include "pycore_blocks_output_buffer.h"
+
+/* On success, return value >= 0
+   On failure, return -1 */
+static inline Py_ssize_t
+Buffer_InitAndGrow(_BlocksOutputBuffer *buffer, Py_ssize_t max_length,
+                   char **next_out, uint32_t *avail_out)
+{
+    Py_ssize_t allocated;
+
+    // The maximum block size accepted by the lib is UINT32_MAX
+    assert(OUTPUT_BUFFER_MAX_BLOCK_SIZE <= UINT32_MAX);
```

Comment:
these are known at compile time right?

turn this into a compile error instead.

```
#if OUTPUT_BUFFER_MAX_BLOCK_SIZE > UINT32_MAX
#error "... some message ..."
#endif
```